src/pages/home/home.tsx

```diff
@@ -1,46 +1,42 @@
-import { ReactNode, useEffect, useState } from 'react';
+import { ReactNode, useState } from 'react';
 
-import ActionBar from '~pages/home/components/ActionBar/ActionBar';
+import isEmpty from 'lodash.isempty';
 
-import { useSocialMediaStore } from '~stores/useSocialMediaStore';
-import isEmpty from '~utils/isEmpty/isEmpty';
+import { useSocialMediaStore } from '~stores/useSocialMediaStore/useSocialMediaStore';
 
 import ComposerEditor from '~components/ComposerEditor/ComposerEditor';
 import MainComposer from '~components/ContentEditor/ContentEditor';
 import FeedbackError from '~components/FeedbackError/FeedbackError';
 import Tabber from '~components/Tabber/Tabber';
 
+import ActionBar from './components/ActionBar/ActionBar';
 import Header from './components/Header/Header';
 import Sidebar from './components/Sidebar/Sidebar';
 
 import scss from './home.module.scss';
 
 function Home(): ReactNode {
-  const { accounts, getAllAccounts } = useSocialMediaStore();
+  const { accounts } = useSocialMediaStore();
   const [isOpen, setIsOpen] = useState(true);
   const [inputText, setInputText] = useState('');
 
   const editor = <ComposerEditor onChange={setInputText} value={inputText} />;
 
-  useEffect(() => {
-    if (isEmpty(accounts)) getAllAccounts();
-  }, [accounts, getAllAccounts]);
-
   return (
     <>
       <Header />
       <main className={scss.content}>
         <aside className={scss.aside}>
           <Sidebar />
         </aside>
-        <div className={scss.aditor}>
+        <div className={scss.editor}>
           <MainComposer
             editor={editor}
             isOpen={isOpen}
             onToggle={() => setIsOpen(!isOpen)}
             title="Main Content"
           />
-          {!isEmpty(accounts) && <Tabber />}
+          {accounts.data && !isEmpty(accounts.data) && <Tabber />}
```

Comment:
Não é necessário accounts.data, o IsEmpty já verifica 
```suggestion
          {!isEmpty(accounts.data) && <Tabber />}
```